reference-architecture/aws-ansible/playbooks/openshift-install.yaml

```diff
@@ -34,6 +41,14 @@
       - vars/main.yaml
   roles:
       - openshift-volume-quota
+- hosts: masters
+  gather_facts: no
```

Comment:
No issues...was just trying to make sure fact gathering was only done on the 1 play that needs it. Any suggestions appreciated